### PR TITLE
fixed healthcheck task

### PR DESCRIPTION
**Pull request by rwlockbg**

### CODE REVIEW

**Review by arminsabouri — DISMISSED** [review]
ApproachACK

**Comment by arminsabouri** [comment]
@rwlockbg I think the block builder int. is experiencing the same bug you described locally. PBFT doesnt seem to be kicking in. Rather it seems to not submit any txs. Need to debug further

**Review by scottmillner — DISMISSED** [review]

File: test_runner.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -o pipefail
 
 # Define the list of strings
 # temporarily removed "e2e_peer_disconnect" since healthcheck service is disabled
```
Comment (scottmillner) [inline test_runner.sh]: nit: remove comment

Comment (rwlockbg) [reply test_runner.sh]: removed, thanks

**Comment by rwlockbg** [comment]
@0xBEEFCAF3 @scottmillner I have modified the integration test now so it physically disconnects the peers upon request. Previously it was just removing them from the peers list. Since I am not sure how to reconnect with them now (the network handler does not have such a command), I have asked Matthias from Reth for help.

**Review by arminsabouri — APPROVED** [review]
tACK 3d58d8d372b6ae21e577596f06e86e66b694798b

**Pull request merged**